src/main/index.ts

```diff
@@ -1,58 +1,42 @@
-import { BrowserWindow, app } from "electron";
+import "reflect-metadata";
+import { Container, injectable } from "inversify";
+import entities from "./entities";
+import AppService from "./services/app.service";
 
-let mainWindow: BrowserWindow | null;
-const winURL =
-  process.env.NODE_ENV === "development"
-    ? `http://localhost:9080`
-    : `file://${__dirname}/index.html`;
-
-function createWindow() {
-  /**
-   * Initial window options
-   */
-  mainWindow = new BrowserWindow({
-    height: 563,
-    useContentSize: true,
-    width: 1000
-  });
-
-  mainWindow.loadURL(winURL);
+/**
+ * Create the DI Container
+ *
+ * @type {Container}
+ */
+let container = new Container();
 
-  mainWindow.on("closed", () => {
-    mainWindow = null;
-  });
+/**
+ * Bind all entities into the DI Container
```

Comment:
Replace "DI" with "dependency injection", since not everyone is familiar with the concept and that's much easier to run a Google search for.